client/src/components/search/searchBar/SearchSuggestion.js

```diff
@@ -14,6 +14,7 @@ const Suggestion = ({ hit, handleMouseEnter, handleMouseLeave }) => {
       href={hit.url}
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
+      target={hit.url && '_blank'}
```

Comment:
This should just be target='_blank', I think? The url is above, in the `href` attribute.

Reply:
it is to check if the hit.url exists, if it does not the target will be null and will not open in a different page, its like an extra step to check if we want to even have the target="_blank" on the a tag :)

Reply:
@michaelparkadze I don't think we need the link to conditionally open in a new tab, as we want the links to have consistent behavior. Also, I believe that each search result should already contain a valid URL. But if we do want to have a check for the truthiness of the URL, then we should consider hiding the result entirely if URL is null, rather than letting the result being unresponsive to user's click. Though this is not in the scope of the issue we are trying to address.

Reply:
So how can we continue with this? I can check if the behavior of the search results are constant and if so just remove the condition or make the necessary changes for them to be consistent. So just let me know how to continue as I am looking to contribute ;)

Reply:
I think the validation on the URL or search results can be left out to keep the PR in scope. This means setting `target` to `_blank` should be enough 🙂

Reply:
@huyenltnguyen I made the necessary changes ;)